### PR TITLE
IMDB templates

### PR DESCRIPTION
**Pull request by Urmish**
Apart from the usual templates, added an adversarial example too

### CODE REVIEW

File: templates/imdb/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    reference: ''
+  b93b74ac-fe95-40b4-9610-318b46ab820f: !Template
+    id: b93b74ac-fe95-40b4-9610-318b46ab820f
+    jinja: '''{{text}} What is the sentiment expressed in this text? ||| {{ ["negative",
+      "positive"] [label] }}'''
+    name: imdb_1
+    reference: ''
+  bd82ba0f-01d4-4fa1-bf8d-07e392c00cd9: !Template
+    id: bd82ba0f-01d4-4fa1-bf8d-07e392c00cd9
+    jinja: '{{text}} Did the reviewer enjoy the movie? ||| {{ ["Yes", "No"] [label]
```
Comment (stephenbach) [inline templates/imdb/templates.yaml:35]: Labels are backwards.

File: templates/imdb/templates.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  c70d1687-2421-49a2-9553-91b8bac4cfbe: !Template
+    id: c70d1687-2421-49a2-9553-91b8bac4cfbe
+    jinja: '{{text}} What is the sentiment expressed by the reviewer for the movie?
+      ||| {{ ["negative", "positive"] [label] }}'
+    name: imdb_4
+    reference: ''
+  e01970ab-42c0-4e6e-a08f-4940d889ef37: !Template
+    id: e01970ab-42c0-4e6e-a08f-4940d889ef37
+    jinja: '{{text}}How does the reviewer feel about the movie? ||| {{ ["They loved
+      it!", "They din''t like it"] [label] }}'
```
Comment (stephenbach) [inline templates/imdb/templates.yaml:48]: Labels are backwards.
`din''t` -> `didn't`

File: templates/imdb/templates.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    reference: ''
+  96538f30-f2c1-430e-8fc6-936a16966d9c: !Template
+    id: 96538f30-f2c1-430e-8fc6-936a16966d9c
+    jinja: '{{text}} What sentiment does the writer express for the movie? ||| {{
+      ["negative", "positive"] [label] }}'
+    name: imdb_7
+    reference: ''
+  af51297c-38a3-4d6c-a8b5-04b1243d7443: !Template
+    id: af51297c-38a3-4d6c-a8b5-04b1243d7443
+    jinja: '{{text}}The sentiment expressed for the movie is ||| {{ ["negative", "positive"]
```
Comment (stephenbach) [inline templates/imdb/templates.yaml:23]: Missing a space between `{{text}}` and the next word.

File: templates/imdb/templates.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      [label] }}'
+    name: imdb_3
+    reference: ''
+  b93b74ac-fe95-40b4-9610-318b46ab820f: !Template
+    id: b93b74ac-fe95-40b4-9610-318b46ab820f
+    jinja: '''{{text}} What is the sentiment expressed in this text? ||| {{ ["negative",
+      "positive"] [label] }}'''
```
Comment (stephenbach) [inline templates/imdb/templates.yaml:30]: Not sure if it's some kind of Jinja escaping thing or something entered in the app, but these render with extra single quotes around the prompt and output.

**Review by stephenbach — CHANGES_REQUESTED** [review]
Awesome, thank you! Just a few things that need to be cleaned up.

File: templates/imdb/templates.yaml (hunk 2 above)
Comment (Urmish) [reply templates/imdb/templates.yaml:48]: I think this is how its stored in the templates file. The prompt is correct as seen here -
image

Comment (stephenbach) [reply templates/imdb/templates.yaml:48]: I think it's rendering ok. There are a few issues though. By "labels backwards" I mean positive is mapped to "they didn't like it." Didn't is misspelled. And there should be a space between `{{text}}` and the question.

Comment (Urmish) [reply templates/imdb/templates.yaml]: Ah. Ok. Fixed it. Chat lingo habits - dint is more easy to write that didnt :).

**Comment by stephenbach** [comment]
Awesome, thank you! Please mark as completed on the spreadsheet!

**Pull request merged**